plugins/coverbrowser.koplugin/main.lua

```diff
@@ -330,36 +478,52 @@ function CoverBrowser:addToMainMenu(menu_items)
                     },
                     {
                         text = _("Show number of pages left to read"),
-                        enabled_func = function() return not BookInfoManager:getSetting("hide_page_info") end,
+                        enabled_func = function()
+                            return not BookInfoManager:getSetting("hide_page_info") and fc.display_mode_type == "list"
+                        end,
                         checked_func = function() return BookInfoManager:getSetting("show_pages_left_in_progress") end,
                         callback = function()
                             BookInfoManager:toggleSetting("show_pages_left_in_progress")
                             fc:updateItems(1, true)
                         end,
                     },
+                    {
+                        text = _("Show file metadata"),
+                        enabled_func = function()
+                            return fc.display_mode_type == "list"
+                        end,
+                        checked_func = function()
+                            return not BookInfoManager:getSetting("hide_file_info")
+                        end,
+                        callback = function()
+                            BookInfoManager:toggleSetting("hide_file_info")
+                            fc:updateItems(1, true)
+                        end,
+                        separator = true,
+                    },
                 },
             },
             {
-                text = _("Display hints"),
+                text = _("Visual cues"),
                 sub_item_table = {
                     {
-                        text = _("Show hint for books with description"),
+                        text = _("Show visual cue for books with descriptions"),
                         checked_func = function() return not BookInfoManager:getSetting("no_hint_description") end,
                         callback = function()
                             BookInfoManager:toggleSetting("no_hint_description")
                             fc:updateItems(1, true)
                         end,
                     },
                     {
-                        text = _("Show hint for book status in history"),
+                        text = _("Show visual cue for books in history tab"),
```

Comment:
This is incorrect text change.

Reply:
I think I finally figure out what these two (this and the next) options do.

I always assumed the dog ear meant book open. I didn't realised it had different meanings across the ui, that is probably not a good thing.

will change the texts though